client/types/disputes.d.ts

```diff
@@ -56,6 +69,7 @@ export interface Dispute {
 	currency: string;
 	created: number;
 	balance_transactions: BalanceTransaction[];
+	payment_intent: string;
```

Comment:
This was missing from the Dispute TS interface, but is a property of the Stripe API dispute object: https://stripe.com/docs/api/disputes/object#dispute_object-payment_intent.